pkg/api/api.go

```diff
@@ -259,6 +259,10 @@ func (hs *HTTPServer) registerRoutes() {
 			apiRoute.Group("/storage", hs.StorageService.RegisterHTTPRoutes)
 		}
 
+		if hs.Features.IsEnabled(featuremgmt.FlagPanelTitleSearch) {
+			apiRoute.Group("/", hs.SearchV2HTTPService.RegisterHTTPRoutes)
```

Comment:
```suggestion
			apiRoute.Group("/search-v2", hs.SearchV2HTTPService.RegisterHTTPRoutes)
```
maybe? better to see the main route in the api file.

i would still like to keep the structure+response "private" until we settle on the whole document storage model

Reply:
> maybe? better to see the main route in the api file.

yeah that looks better, moved